### ISSUE TITLE
General comments & typos

### ISSUE THREAD

**Issue by mandyWW** [buggyrace/buggy-race-server#200]
- Unclear that you need to redeploy when changing config e.g. https://rhul.buggyrace.net/admin/settings/github - would be helpful to have a warning at the top of each page that you need to do this rather than assume it will pick up immediately.

**Comment by mandyWW** [comment]
- Link https://rhul.buggyrace.net/tech_notes/localhost does not work

**Comment by mandyWW** [comment]
Can we have anchor element for the buy server poster page so it's easier to link to parts of the poster documentation? I feel it's not clear that they need to record their progress as they go along so wanted to link to the section "Storing texts on this race server".

**Comment by davewhiteland** [comment]
> * Unclear that you need to redeploy when changing config e.g. https://rhul.buggyrace.net/admin/settings/github - would be helpful to have a warning at the top of each page that you need to do this rather than assume it will pick up immediately.

This behaviour is configurable: if `IS_SHOWING_RESTART_SUGGESTION` is set to `Yes` (in the "Server" group of config settings) then a warning is shown. The reason why it's not automatically on is we didn't seem to need restart for changes to pick up, but this was never investigated thoroughly (it was unexpected, because originally I assumed a redeploy would always be needed). Really need to determine when a redeploy is needed (or if it always is).

**Comment by davewhiteland** [comment]
> Can we have anchor elements for the headings in the buggy server poster page so it's easier to link to parts of the poster documentation? 

Yes, there should be anchor elements on **all** headings on the server pages: see #154

**Comment by davewhiteland** [comment]
> Would also be useful to make this very clear on the main tasks page and at the end of every task else there is a risk people won't log in and won't record their progress.

Yup, especially as `IS_STORING_STUDENT_TASK_TEXTS` is a config setting, so the server could add a message saying "remember to record how you approached or solved this in [a task text](#)" _[linking to the task text input]_ to the task descriptions (so it doesn't need to be added to each task text manually)

**Comment by davewhiteland** [comment]
> * Link to .../tech_notes/localhost does not work

oof yes it should be `tech-notes/localhost` (hyphen not underscore) — looks like it's only wrong when linking to the page about localhost (dunno why it's only that one), in about 9 places (!)

**Comment by mandyWW** [comment]
Task 1-TEMPLATE typo "It's up to you What data from the specs do want to display"

**Comment by mandyWW** [comment]
Type in Buggy Server - "Helful hints to help students understand the problem and perhaps concepts around it"
